### ISSUE TITLE
SLAYER - CLEFT IN TWAIN Dealing more damage that it should

### ISSUE THREAD

**Issue by ojoaop** [WarEmu/WarBugs#19759]
**Expected behavior and actual behavior:**
Cleft In Twain should deal a constant damage, Tick damage or Tick damage multiplied by critical damage. It occurs when you aren't using the tactic Fierce Might.

When you are using this tactic Fierce Might, your critical damage with this ability is to much higher in each ability tick, as you can see on the video.

**Steps to reproduce the problem:**
1 - Equip the tactic Fierce Might and the skill Cleft In Twain.
2 - Put your own life at below 90%, 80% ..., ... and 10% (where the bug is more visible)
3 - Check combat log or the damage addon.

**Testing Screenshots/Videos/Evidences:** [https://youtu.be/uahNuIN1sOk](https://youtu.be/uahNuIN1sOk)

The video is more explicit from time 0:55

**Comment by lilimx** [comment]
Hey,

could you test this on the training construct in Altdorf aswell?

**Comment by ojoaop** [comment]
Hey! Sure, I can

Here is: [https://youtu.be/oqNQm7Z3GWY](https://youtu.be/oqNQm7Z3GWY)

I did the test with a friend as well, at Garden QU'APH, but didn't record it. But the behavior is the same.

Adittionaly I receive messages like this (Yellow):
![Screenshot_6](https://user-images.githubusercontent.com/97126301/150853373-6cad6fa4-df26-43d5-8171-47682e81980e.png)

**Comment by Bejkon** [comment]
Hej,

This has nothing to do with CiT specifically.

Its Fierce Might that is bugged. I just havent been bothered to report it myself, since it pretty clear to see if the chat warnings ever actually got looked at. Anyways, since you brought it up, here we go.

There are some report( https://github.com/WarEmu/WarBugs/issues/15925 ) of the chat msg that shows up, this report seem to think its the msg thats shown wrongfully. In reality the crit values are bonkers, but not always nor nessesarily easily noticed.

From my testing, this is how and what is bugged:

Fierce Might grants 10% crit dmg for every 10% hp missing. This is generally working as intended for any DD hit.

However, for dots, this additional crit multiplier keeps stacking for each consecutive tick. This applies to all dots. Keep in mind ID is not a traditional dot and thus is not affected.

So for example, 

If the slayer is at 50% hp, and uses Relentless Strike:

Base CritMultiplier= 1.35-1.55 rng
Tick 1 Multiplier; Base+0.5 = 1.85-2.05 (as intended with the tactic)
Tick 2 Multiplier; T1+0,5 = 2.35-2.55
Tick 3 Multiplier; T2+0,5 = 2,85-3.05

Now this wouldnt trigger any chat warnings, but its still affected by the stacking bug and easily tested.

Now, if we do the same calc but for a slayer at 10% hp and using Cleft in Twain which has 6 ticks instead of 3 we end up with:

T6 multiplier; Base+(0.9*6)=base+5.4 CritModRangeT6=5.75-5.95.

In other words this case can produce a 595% crit multiplier for the 6th tick.

Now you could go even lower in HP and make the modifier higher, but you get my point.

There you have it, thats the bug.

Why hasnt it been abused/noticed/fixed since its been around for years? 
It probably has to do with CiT being subpar its competition masterywise and even more rarely ran in combo with FM. ID not being affected surely made the bug less noticable aswell. You have to be extremely low HP to trigger the warnings with Relentless Strike(3 ticks) and decently low with Onslaught(4 ticks) and ofc get crits on specifically the later ticks.

Good find tho.

/Bejk

**Comment by lilimx** [comment]
Hey,

so Bejkon added your report already to the related ticket. So Im going to close this one.
